scripts/build_shapes.py

```diff
@@ -805,15 +875,26 @@ def gadm(
     out_logging = snakemake.config["build_shape_options"]["out_logging"]
     year = snakemake.config["build_shape_options"]["year"]
     nprocesses = snakemake.config["build_shape_options"]["nprocesses"]
+    contended_areas = snakemake.config["build_shape_options"]["contended_areas"]
     EEZ_gpkg = snakemake.input["eez"]
     worldpop_method = snakemake.config["build_shape_options"]["worldpop_method"]
     gdp_method = snakemake.config["build_shape_options"]["gdp_method"]
     geo_crs = snakemake.config["crs"]["geo_crs"]
     distance_crs = snakemake.config["crs"]["distance_crs"]
 
-    country_shapes = countries(countries_list, geo_crs, update, out_logging)
+    keep_all_areas = False if contended_areas == "drop" else True
```

Comment:
I'd pass the contended_areas option directly to the function instead of parsing it to boolean; it gives more flexibility as discussed above.

Reply:
Revised